src/directory/singleUser.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "../directory/person.css";
+
+class Person extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div className="column">
```

Comment:
@Alihaider168 let's name this class as `member-preview` or `avatar` or `member-card` or something.